app/models/user_target.rb

```diff
@@ -3,6 +3,13 @@ class UserTarget < ApplicationRecord
   belongs_to :topic
 
   validates :title, :user, :topic, :latitude, :longitude, :area, presence: true
+  validate :check_target_limit
```

Comment:
I would name it `target_limit` so it sounds more readable when you are seeing the code: "validate target limit" instead of "validate check target limit". I dont know if I'm clear in what's my point.